ee/app/livechat-enterprise/server/settings.js

```diff
@@ -36,5 +36,30 @@ export const createSettings = () => {
 		enableQuery: { _id: 'Livechat_waiting_queue', value: true },
 	});
 
+	settings.add('Livechat_auto_close_inactive_rooms', false, {
+		type: 'boolean',
+		group: 'Omnichannel',
+		section: 'Sessions',
+		i18nLabel: 'Enable_omnichannel_auto_close_inactive_rooms',
+		i18nDescription: 'Enable_omnichannel_auto_close_inactive_rooms_description',
+	});
+
+	settings.add('Livechat_auto_close_inactive_rooms_action', 'close', {
```

Comment:
As I mentioned above, the `close` will be the only action now, so we don't need this setting, we can rely on `Livechat_auto_close_inactive_rooms`